src/XAMLTools.Core/XAMLCombine/XAMLCombiner.cs

```diff
@@ -37,21 +46,24 @@ public void Combine(string sourceFile, string targetFile)
             // Load resource file list
             var resourceFileLines = File.ReadAllLines(sourceFile);
 
-            this.Combine(resourceFileLines, targetFile);
+            this.Combine(resourceFileLines, targetFile, importMergedResourceDictionariesReferences);
         }
 
         /// <summary>
         /// Combines multiple XAML resource dictionaries in one.
         /// </summary>
         /// <param name="sourceFiles">Source files.</param>
         /// <param name="targetFile">Result XAML filename.</param>
-        public string Combine(IEnumerable<string> sourceFiles, string targetFile)
+        /// <param name="importMergedResourceDictionariesReferences"></param>
+        public string Combine(IEnumerable<string> sourceFiles, string targetFile, bool importMergedResourceDictionariesReferences = false)
```

Comment:
remove default parameter value so we can never forget to pass it in?

Reply:
Yeah, let's do this, better design